### PR TITLE
Add note about config for docs endpoints in readme

### PR DESCRIPTION
**Pull request by paulineribeyre**
### Improvements
- Add note about configuration for local docs endpoint in readme

### Dependency updates
- fastapi to 0.65.*
- explicitly make pydantic a dependency
- pin cryptography to 3.1 to avoid build issues with cryptography 3.4.7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,4 +64,4 @@ OR
 uvicorn requestor.asgi:app --reload
 ```
 
-Try out the API at: <http://localhost:8000/docs>.
+Try out the API at <http://localhost:8000/_status> or <http://localhost:8000/docs> (you might have to set `DOCS_URL_PREFIX` to `""` in your configuration file for the docs endpoint to work).
```
Comment (nss10) [inline README.md:67]: 1. We don't get the API when we hit [<http://localhost:8000/_status>](url) right? I only see a `{status:OK}` message. 
2.  > you might have to set `DOCS_URL_PREFIX` to `""`

    Why is it **might** ? Is it not going to be the same all the time?

Comment (paulineribeyre) [reply README.md:67]: _status is an endpoint of the API

Comment (paulineribeyre) [reply README.md:67]: 2. it used to work without it, and we didn't make changes, so i'm not sure what caused it. i don't have time to investigate but just updating the readme might help the next person

**Review by nss10 — APPROVED** [review]
LGTM

**Review by nss10 — APPROVED** [review]

**Pull request merged**